packages/fxa-react/lib/storybooks.tsx

```diff
@@ -17,3 +22,15 @@ export const withLocalization: Decorator = (Story) => (
     <Story />
   </AppLocalizationProvider>
 );
+
+export const withLocation: (location: string) => Decorator =
+  (location) => (Story) => {
+    const source = createMemorySource(location);
+    const history = createHistory(source);
+
+    return (
+      <LocationProvider history={history}>
```

Comment:
We have a lot more stories with a simple `<LocationProvider>` (with no `history` passed) in than not, so should we make the `location` argument optional? I'm fine either way, this is great to have!